inc/template-tags.php

```diff
@@ -35,7 +35,7 @@ function twentytwenty_site_logo( $args = array(), $echo = true ) {
 			'title_class' => 'site-title',
 			'home_wrap'   => '<h1 class="%1$s">%2$s</h1>',
 			'single_wrap' => '<div class="%1$s faux-heading">%2$s</div>',
-			'condition'   => is_front_page() || is_home(),
+			'condition'   => ( is_front_page() && is_home() ) || ! is_singular(),
```

Comment:
@mukeshpanchal27 & @acosmin Currently, the following pages still show two `<h1>`:

- Author pages, e.g. http://wp.test/author/themedemos/
- Category pages, e.g. http://wp.test/category/markup/
- Tag pages, e.g. http://wp.test/tag/css/

I've tested this PR using [Screaming Frog SEO Spider](https://www.screamingfrog.co.uk/seo-spider/) which is available both as a free and a paid version. The free version should be more than sufficient for theme testing purposes. 😉